server/src/main/java/org/candlepin/dto/api/v1/GuestIdDTO.java

```diff
@@ -0,0 +1,216 @@
+/**
+ * Copyright (c) 2009 - 2017 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.dto.api.v1;
+
+import org.candlepin.common.jackson.HateoasArrayExclude;
+import org.candlepin.common.jackson.HateoasInclude;
+import org.candlepin.util.MapView;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+
+import io.swagger.annotations.ApiModel;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlRootElement;
+
+/**
+ * A DTO representation of the GuestId entity
+ */
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.PROPERTY)
+@ApiModel(parent = TimestampedCandlepinDTO.class, description = "DTO representing a consumer capability")
+@JsonFilter("GuestFilter")
+public class GuestIdDTO extends TimestampedCandlepinDTO<GuestIdDTO> {
+    public static final long serialVersionUID = 1L;
+
+    protected String id;
+    protected String guestId;
+    protected Map<String, String> attributes;
+
+    /**
+     * Initializes a new GuestIdDTO instance with null values.
+     */
+    public GuestIdDTO() {
+    }
+
```

Comment:
Probably a good idea to add a comment saying what this constructor is used for (e.g. tests).

Reply:
I assume it was the other constructor. ill add the comment.